### PR TITLE
feat: adding ability to remove favorites to favorites page

### PR DESCRIPTION
**Pull request by bcastillo32**
I was looking to add the already existing favorites section (components/pets/Favorites/FavoriteSection.tsx) to the animal card. 

There looks to still be a bug that is affecting adding/removing from favorites that existed before I pushed this 

so I'm not sure if I did something wrong or the bug is having an impact.

### CODE REVIEW

File: src/components/layout/PetCard.tsx [hunk 1]
```diff
@@ -44,6 +45,7 @@ export default function PetCard(props: PetCardType) {
           <LinkContainer to={`/animal/${id}`}>
             <Button className="card__btn">More Info</Button>
           </LinkContainer>
+          <FavoriteSection id={id} />
```
Comment (redxzeta) [inline src/components/layout/PetCard.tsx:48]: Home page uses this petcard  too and all the users access it. 

You can Add a button in `Favorites.tsx` where pet List map is 
```
          {petList.map((pet) => (
            <><PetCard
              key={pet.id}
              breeds={pet.breeds}
              id={pet.id}
              name={pet.name}
              photos={pet.photos}
              type={pet.type}
              primary_photo_cropped={pet.primary_photo_cropped}
            />  
// some button with remove favorites functionality
</>
          ))}
```

Comment (redxzeta) [reply src/components/layout/PetCard.tsx:48]: the remove button onlclick would have something similar to function

```
  const removeFav = async () => {
    await executeDelete((query) => query.eq("id", removalId), {
      returning: "minimal",
      count: "estimated",
    });
    dispatch(removeFavoritePet(removalId));
  };
```

**Review by redxzeta — CHANGES_REQUESTED** [review]
Some feedback

File: package.json [hunk 2]
```diff
@@ -19,6 +19,9 @@
     "bootstrap-icons": "^1.8.1",
     "bootswatch": "^5.1.3",
     "jwt-decode": "^3.1.2",
+    "list": "^2.0.19",
+    "node": "^17.7.2",
+    "nvm": "0.0.4",
```
[the remark this answers is not in the document]
Comment (bcastillo32) [reply package.json:24]: > not sure what these are for

I had to install a newer version of node and was trying a couple of different things before I got it to work. I was getting an error about needing a more recent node version so tried using nvm (node version manager)..not sure what "list" is

[the remark this answers is not in the document]
Comment (redxzeta) [reply package.json:24]: u normally dont install `node` or `nvm` on `package.json`

you can download the latest node version here https://nodejs.org/en/ current requirements node>16

nvm you can follow instructions here https://github.com/nvm-sh/nvm#installing-and-updating tho doesn't work on windows i believe

[the remark this answers is not in the document]
Comment (redxzeta) [reply package.json:24]: after that u can just remove these from the `package.json`

[the remark this answers is not in the document]
Comment (bcastillo32) [reply package.json]: > after that u can just remove these from the `package.json`

took care of this

**Review by redxzeta — APPROVED** [review]

**Pull request merged**